tcp_rr.c

```diff
@@ -203,14 +208,19 @@ static void run_client(struct thread *t)
         struct flow *stop_fl;
         int epfd, i;
         char *buf;
+        CLEANUP(free) int *client_fds;
```

Comment:
With cleanup annotations you usually initialize the pointer to `NULL` so that we don't accidentally `free()` garbage if a return point exists before the allocation where we assign the pointer. It doesn't matter here but I would add it so that we don't leave a "trap" for ourselves to fall into later.